pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[build-system]
+# To get a reproducible wheel, wheel must be pinned to the same version as in
+# dls-python3, and setuptools must produce the same dist-info. Cap setuptools
+# to the last version that didn't add License-File to METADATA
+requires = ["setuptools<57", "wheel==0.33.1"]
```

Comment:
Can ditch this now
```suggestion
requires = ["setuptools>=45", "setuptools_scm[toml]>=6.2", "wheel"]
```

Reply:
Although not pinning wheel stops us making a reproducible wheel. But do we care?